### PR TITLE
Documentation for angular runtime

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
This PR adds code snippets for the angular runtime for: 
- Quick Start
- Layout
- Playback
- State Machine

I used the same coding style and exemple file as in Web & Flutter snippet.
Unfortunately I didn't know how to test it, so I hope it displays as expected. Let me know if I need to change something.

Best

### CODE REVIEW

File: runtimes/state-machines.md [hunk 1]
```diff
@@ -198,6 +206,45 @@ void _onStateChange(
...
+  </riv-state-machine>
+</canvas>
+
+<input type="radio" formControl="level" value="0"> Car
+<input type="radio" formControl="level" value="1"> Train
+<input type="radio" formControl="level" value="2"> Airplane
+```
+_The `stateChange` output will display the list of state changed during the same frame._
+
+### Tigger Input
```
Comment (avivian) [inline runtimes/state-machines.md:235]: typo
```suggestion
### Trigger Input
```

File: runtimes/state-machines.md [hunk 2]
```diff
@@ -198,6 +206,45 @@ void _onStateChange(
...
+If the input is a trigger you can access it with the export as `rivInput`:
+```html
+<canvas riv="vehicles">
+  <riv-state-machine name="bumpy" play>
+    <riv-input #trigger="rivInput" name="bumpt" (change)="showInput($event)"><riv-input>
+  </riv-state-machine>
+</canvas>
+
+<button (click)="tigger.fire()">Bumpt</button>
```
Comment (avivian) [inline runtimes/state-machines.md:244]:
```suggestion
<button (click)="trigger.fire()">Bumpy</button>
```

File: runtimes/state-machines.md [hunk 3]
```diff
@@ -198,6 +206,45 @@ void _onStateChange(
...
+<input type="radio" formControl="level" value="2"> Airplane
+```
+_The `stateChange` output will display the list of state changed during the same frame._
+
+### Tigger Input
+If the input is a trigger you can access it with the export as `rivInput`:
+```html
+<canvas riv="vehicles">
+  <riv-state-machine name="bumpy" play>
+    <riv-input #trigger="rivInput" name="bumpt" (change)="showInput($event)"><riv-input>
```
Comment (avivian) [inline runtimes/state-machines.md:240]:
```suggestion
    <riv-input #trigger="rivInput" name="bumpy" (change)="showInput($event)"><riv-input>
```

**Comment by GrandSchtroumpf** [comment]
I'm terribly sorry for the typos. I'm updating it right away

**Review by avivian — APPROVED** [review]
This is awesome, thanks for spending the time to add this! 

Just a few typos to fix up and we'll get this merged today.

**Comment by GrandSchtroumpf** [comment]
Typo fixed. Thanks for the quick review ❤️

**Pull request merged**